pages/project/[id].js

```diff
@@ -95,3 +161,58 @@ const DeleteButton = styled.button`
   height: 25px;
   width: 30px;
 `;
+
+const NotesForm = styled.form`
+  display: flex;
+  position: fixed;
+  bottom: 5px;
+  display: flex;
+  position: fixed;
+  bottom: 3.7em;
+  width: 100%;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  flex-direction: column;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+`;
+
+const NotesCounter = styled.p`
+  position: absolute;
```

Comment:
Here you should also think about if `position: absolute` is really necessary. If so you should declare `position: relative` for your parent container.